chart/templates/grafana-datasources-sec.yaml

```diff
@@ -31,9 +31,11 @@ stringData:
 {{ if $tsEnabled -}}
 {{- $isDefault := not $promEnabled -}}
 {{- $root := . -}}
+{{- $host := tpl .Values.grafana.timescale.database.host $root -}}
+{{- $port := index .Values "timescale-prometheus" "connection" "port" | int -}}
```

Comment:
Added, but we can't do it the same way as we did the hostname, since the port is an int. Helm conventions say not to put quotes around ints that arent environment variables, so we will have to just edit the port number in 3 places.